### ISSUE TITLE
CI / signup

### ISSUE THREAD

**Issue by prk-dismoi** [dis-moi/contribution-interface#1]
test

**Comment by prk-dismoi** [comment]
As a user, I would like to be able to sign up on the Dismoi website in order to create a contributor account and start posting contributions from that account.

**Comment by prk-dismoi** [comment]
@lutangar can you check the above when you have a sec. Missing something? Too much detail? not enough? lmk tx.

**Comment by lutangar** [comment]
:scream: Way too many details!

You're describing a lot of little things about UI implementation details here, mockup are sufficient for the most part.
... but I think we are missing something mobile centric here, I'd love to see an app looking design instead of just a responsive website. Might not be your job tho'

Some of the behavior you describe, for example validation, may vary depending on the choice of library made.

> What do you think @newick ?

We also spoke about optional stuff and when I see things like the little eye for the password are typically optional.
I hope no one will follow this to the letter everything here.

On the other hand...
About CGU is this an existing page (dismoi.io) or this content that should be written by someone?

**Comment by prk-dismoi** [comment]
Cool, thanks for feedback.

- detail/validation: roger that, will trim. Do you still need some copy for error handling?
- mobile: this is only functional, Marco and Erick will mobilize
- optional eye: makes sense. Will need your guidance on these things in general, I dont always know how long some tasks take. Next question is do we dissassociate these things and create tickets to deal with later. I say yes.
- yep CGU is an existing page: https://www.dismoi.io/cgu/

**Comment by newick** [comment]
> Some of the behavior you describe, for example validation, may vary depending on the choice of library made.
Yep, it'd depend of the feedback we'd need to send to the users. But I think it's a nice thing to get the idea written.

**Comment by lutangar** [comment]
@newick I'd like your opinion on the initial description of this feature by @prk-dismoi . 
We're trying to find the sane level of details for each role: Project, Integration and Dev
I also think it's important to keep focus on the main features to keep a descent celerity and move down less important things down the pile of priority. I insist, it doesn't mean it won't be done, but just later on.

**Comment by prk-dismoi** [comment]
Trello Design ticket: https://trello.com/c/XnP5KbAU/272-signup

**Comment by newick** [comment]
![image](https://user-images.githubusercontent.com/657454/112647705-13bf6800-8e49-11eb-9338-4baa14c00f74.png)
looks good, right?
